script.js

```diff
@@ -6,6 +6,12 @@ const rootElem = document.getElementById('root');
 const searchBox = document.getElementById('search');
 const searchCount = document.getElementById('search-count');
 
+const seasonAndEpisode = (season, episode) => {
+  season = season.toString().padStart(2, '0');
+  episode = episode.toString().padStart(2, '0');
+  return `S${season}E${episode}`;
+};
+
```

Comment:
I like how you've separated this block of code into its own function and you're using it in different parts of your code.  This gives me ideas.